tla/consensus/ccfraft.tla

```diff
@@ -656,6 +656,7 @@ ChangeConfigurationInt(i, newConfiguration) ==
     \* Keep track of running reconfigurations to limit state space
     /\ reconfigurationCount' = reconfigurationCount + 1
     /\ removedFromConfiguration' = removedFromConfiguration \cup (CurrentConfiguration(i) \ newConfiguration)
+    /\ \A addedNode \in (newConfiguration \ CurrentConfiguration(i)) : nextIndex' = [nextIndex EXCEPT ![i][addedNode] = Len(log[i]) + 1]
```

Comment:
I suggest adding the pointer to https://github.com/microsoft/CCF/blob/a648fc3743a77e363b8d8a3f4e002cc5828d57a3/src/consensus/aft/raft.h#L2401 here.